### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by localized26**
This pull request was initiated by Lokalise (user Roman Tishakov) at 2021-08-25 16:21:48

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -562,18 +577,18 @@
...
             "SRFM": {
-                "comment": "Surf Medal True NFT tokens",
+                "comment": "Surf 메달 True NFT 토큰",
                 "stake": "",
-                "title": "My Surf Medals"
+                "title": "내 Surf 메달"
             }
         },
-        "disableAutoresume": "이력서 자동 생성 비활성화하기",
+        "disableAutoresume": "Auto-resume 기능 끄기",
```
Comment (BigRoma) [inline src/ko.json:588]: @kkkhesha Check this translation

File: src/ko.json [hunk 2]
```diff
@@ -1029,8 +1044,8 @@
             "autoresumeDisabled": "자동 생성 비활성화됨",
             "autoresumeEnable": "자동 생성 활성화하기",
             "autoresumeEnabled": "자동 생성 활성화됨",
-            "autoresumeFee": "자동 추정 수수료",
-            "available": "{0} 를 인출할 수 있습니다.",
+            "autoresumeFee": "Auto-resume 수수료",
```
Comment (BigRoma) [inline src/ko.json:1047]: @kkkhesha And this

File: src/ru.json [hunk 3]
```diff
@@ -211,7 +226,7 @@
                 "creator": "Создано",
                 "donate": "Поддержать",
                 "error": {
-                    "failedToSaveFavorite": "Не удалось обновить статус. Повторите попытку позже"
+                    "failedToSaveFavorite": "Не удалось сохранить любимый статус. Повторите попытку позже"
```
Comment (AnnBoo) [inline src/ru.json]: любимый статус - это точно корректная формулировка?

File: src/ru.json [hunk 4]
```diff
@@ -494,28 +509,28 @@
...
         "migrate": "Перенести кошелек в Surf",
         "name": "Имя",
+        "nameTaken": "Выберите другое имя. Это уже занято",
         "newWalletName": "Кошелек",
         "pair": "Привязать аппаратный кошелек к учетной записи",
         "removeWallet": "Удалить кошелек",
         "restore": "Восстановить кошелек",
-        "addWallet": "Добавить кошелек",
-        "restoreHint": "Surf currently supports only verified native wallets. You will not be able to open any other type of wallet.",
+        "restoreHint": "Вы можете восстановить кошелек, созданный только в Surf",
```
Comment (AnnBoo) [inline src/ru.json]: Переведено не полностью: 
```suggestion
        "restoreHint": "В настоящее время поддерживается только восстановление кошельков, созданных в Surf.",
```

File: src/ru.json (hunk 3 above)
Comment (BigRoma) [reply src/ru.json]:
```suggestion
                    "failedToSaveFavorite": "Не удалось обновить статус. Повторите попытку позже"
```

**Review by BigRoma — APPROVED** [review]

**Review by AnnBoo — APPROVED** [review]

**Review by AnnBoo — APPROVED** [review]

File: src/en.json [hunk 5]
```diff
@@ -104,21 +105,24 @@
...
+            "button": "Network",
             "endpoint": "Endpoint",
-            "endpointHint": "Please cut the number of endpoints to one address"
+            "endpointHint": "Please cut the number of endpoints to one address",
+            "name": "Name",
+            "title": "Network",
+            "titleAdd": "New network",
+            "titleEdit": "Edit network"
+        },
+        "nightTheme": {
```
Comment (sertony) [inline src/en.json:124]: This was removed in Surf

**Review by sertony — CHANGES_REQUESTED** [review]

File: src/en.json [hunk 6]
```diff
@@ -52,8 +52,9 @@
         },
         "separators": {
             "accounts": "Accounts",
+            "collectibles": "Collectibles",
             "investments": "Services",
-            "collectibles": "Collectibles"
+            "tokens": "Tokens"
```
Comment (sertony) [inline src/en.json:57]: Oh, this was also removed

**Review by sertony — CHANGES_REQUESTED** [review]

**Review by sertony — APPROVED** [review]
Discussed that left comments will be resolved once development is synced with lokalise.com

**Pull request merged**